tests_for_plate.py

```diff
@@ -0,0 +1,115 @@
+import unittest
+from plate import PlateCell
+from plate import Dimensions
+
+class TestPlateCell(unittest.TestCase):
+    """Tests for PlateCell class"""
+
+    """Tests for 'calculate_row_and_column' method"""
+
+    def test_first_row_and_first_column_are_returned_for_the_first_cell(self):
+        dimensions = Dimensions(16, 24)
+        cell = PlateCell(0, dimensions)
+        row_and_column = cell.calculate_row_and_column()
+        self.assertEqual([0, 0], row_and_column)
+
+    def test_sixteenth_row_and_twelfth_column_are_returned_for_the_middle_cell(self):
+        dimensions = Dimensions(16, 24)
+        cell = PlateCell(191, dimensions)
+        row_and_column = cell.calculate_row_and_column()
+        self.assertEqual([15, 11], row_and_column)
+
+    def test_sixteenth_row_and_twenty_fourth_column_are_returned_for_the_last_cell(self):
+        dimensions = Dimensions(16, 24)
+        cell = PlateCell(383, dimensions)
+        row_and_column = cell.calculate_row_and_column()
+        self.assertEqual([15, 23], row_and_column)
+
+    def test_zero_row_and_zero_column_are_returned_for_0_cell(self):
+        dimensions = Dimensions(16, 24)
+        cell = PlateCell(0, dimensions)
+        row_and_column = cell.calculate_row_and_column()
+        self.assertEqual([0, 0], row_and_column)
+
+    def test_correct_row_and_column_are_returned_for_decimal_cell_number(self):
```

Comment:
"correct" is too vague, It would be vague in manual checklists either. Need something more descriptive which describes _how_ decimal numbers are converted to integer numbers.

But in this case it seems more appropriate to throw an error. We'd never see cell numbers in decimal form.